.github/workflows/GPU.yml

```diff
@@ -0,0 +1,43 @@
+name: GPU
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: [self-hosted, GPU, benchmark]
+
+    steps:
+      - name: cleanup
+        run: rm -rf *
+      - name: Checkout votca repo
+        uses: actions/checkout@v2
+        with:
+          repository: votca/votca
+          submodules: true
+          ref: master
+          path: votca
+          
+      - name: Checkout xtp
+        uses: actions/checkout@v2
+        with:
+          path: votca/xtp
+          
+      - name: Configure Cmake
+        run: cmake -B builddir -DBUILD_CSGAPPS=ON -DBUILD_XTP=ON -DENABLE_TESTING=ON -DCMAKE_BUILD_TYPE=Release -DUSE_CUDA=ON votca
+        
+      - name: Build
+        run: cmake --build builddir --parallel 6
```

Comment:
Where is the 6 coming from?

Reply:
The machine it is running on has 6 cores

Reply:
I would just use `$(nproc)` here.